gobblin-example/src/main/java/gobblin/example/wikipedia/EncryptedEmbeddedWikipediaExample.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gobblin.example.wikipedia;
+
+import avro.shaded.com.google.common.base.Joiner;
+import gobblin.annotation.Alias;
+import gobblin.configuration.ConfigurationKeys;
+import gobblin.publisher.BaseDataPublisher;
+import gobblin.runtime.api.JobTemplate;
+import gobblin.runtime.api.SpecNotFoundException;
+import gobblin.runtime.cli.EmbeddedGobblinCliOption;
+import gobblin.runtime.cli.EmbeddedGobblinCliSupport;
+import gobblin.runtime.cli.PublicMethodsGobblinCliFactory;
+import gobblin.runtime.embedded.EmbeddedGobblin;
+import gobblin.runtime.template.ResourceBasedJobTemplate;
+import gobblin.writer.AvroDataWriterBuilder;
+import gobblin.writer.Destination;
+import gobblin.writer.WriterOutputFormat;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import org.apache.commons.cli.CommandLine;
+
+
+/**
+ * Embedded Gobblin to run Wikipedia example.
+ */
+public class EncryptedEmbeddedWikipediaExample extends EmbeddedGobblin {
```

Comment:
I think it's better to use the same wikipedia example with an encryption option.